demisto_sdk/commands/init/tests/initiator_test.py

```diff
@@ -473,6 +475,60 @@ def test_template_integration_init(monkeypatch, initiator, tmpdir, template):
     assert not diff, f'There\'s a missing file in the copied files, diff is {diff}'
 
 
+def test_integration_init_with_ignore_secrets(initiator, tmpdir, monkeypatch, mocker):
+    """
+    Tests `integration_init` function with a given script template name.
+    Given
+        - An Integration template - HelloWorld.
```

Comment:
Did you test with the other templates?

Reply:
I suggest adding at least one more template just to make sure everything is working as expected.